news/_posts/performance/2020-12-07-performance-and-prestashop.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title:  "Performance and PrestaShop"
+subtitle: "Sorry, Goose, but it's time to buzz the tower."
+date:   2020-12-04 9:00:00
+authors: [ Claude-ArnaudPerrot ]
+icon: icon-airplane
+tags:
+ - performance
+ - whitepaper
+---
+
+Let's start discussing performance and tuning with PrestaShop! 
+
+# Performance (& PrestaShop)
+
+You may have heard of it, but we'll soon release a White Paper, which is all about PrestaShop's performance, how to benchmark it and how it behaves under loads. And what to do to keep it responding under 300ms.
+
+Still, there are a number of matters that won't be covered in this White Paper - hopefully the first in a series, fingers crossed - that we'd love to shine a light on.
+
+Because, funny thing is, we often talk about performance as something we all understand and it is mostly true: we share the same expectations. 
+
+Though, we're not always on line with its implications.
+
+So, here we go!
+
+## Performance is subjective
+
+It may seem counter-intuitive coming from a technical environment, either from web development or any other domain, but still performance is all about capabilities.
+
+Just to illustrate, let's take this well-known number:
+
+![NotANumber](https://upload.wikimedia.org/wikipedia/commons/5/56/Answer_to_Life.png)
+
+Is it high? Or is it low? But compared to what? In which context?
+
+Same thing, if I'm told that in a [GCP](https://cloud.google.com/gcp/) environment, the maximum write sustained IOPS per GB for a regional SSD drive is 30: does it mean it performs? Is it good or...? 
+
+But let's be even more precise: in our White Paper scope, we assumed a web page should load *under* 300ms to be deemed acceptable for our users. Hence, we based our benchmarks on this value - and results above this response time where discarded as unacceptable.
+
+Still, 300ms is not telling us *anything* specific about performance. It's neither good, nor bad. It's all a matter of interpretation, analysis and finding what is *felt* acceptable by our users. 
+
+And sticking to it, of course.
+
+At the end of the day, the suitable performance is the one that fulfills your consummers' expectation.
+
+## There's not "one performance problem"
+
+When you're dealing with a performance issue, you're in fact dealing with "*the currently visible bottleneck*". And as soon as you've managed it, another bottleneck will take its place. Always.
+
+![Botlleneck](https://www.techconsumerguide.com/wp-content/uploads/2019/10/cpu-gpu-bottleneck.jpg)
```

Comment:
```suggestion
![Bottleneck](https://www.techconsumerguide.com/wp-content/uploads/2019/10/cpu-gpu-bottleneck.jpg)
```

Reply:
Could you copy the image (and set the source in the bottom of the article) ?

Reply:
I could, but I wonder about copyrights.
If so, I'd rather make one or have one of our own. What do you think ?

Reply:
In https://build.prestashop.com/news/prestashop-in-2019-and-beyond-part-2-pain-points/ @eternoendless used images hosted on Build with an apparent copyright and/or author mention.

I think if you make it clear that the image does not belong to us it is fine.

Reply:
So, I've just updated the images and included them to the repository.
Let me know what you think ;)